tests/integration/test_integration.sh

```
@@ -34,11 +34,15 @@ test_sync_simulation() {
 
 # Test command consistency
 test_command_consistency() {
-    local commands_in_readme=$(grep -c "^\s*- \`/[a-z]*\`" "$ORIGINAL_DIR/README.md" || echo 0)
-    local commands_in_dir=$(ls "$ORIGINAL_DIR/.claude/commands/"*.md 2>/dev/null | wc -l | xargs)
+    local commands_in_readme=$(grep -c "^\s*- \`/[a-z]*\`" "$ORIGINAL_DIR/README.md" 2>/dev/null || echo 0)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)